pkg/cmd/pulumi/org_search_ai.go

```diff
@@ -119,10 +133,16 @@ func newSearchAICmd() *cobra.Command {
 		&scmd.queryString, "query", "q", "",
 		"Plaintext natural language query",
 	)
-
+	cmd.PersistentFlags().BoolVar(
+		&scmd.openWeb, "web", false,
+		"Open the search results in a web browser.",
+	)
+	cmd.PersistentFlags().StringVar(
```

Comment:
I'd say don't bother with this, lets leave it hardcode to app.pulumi.com and we'll ask service for endpoint to get the frontend from the backend we've got from login.